state/spaces.go

```diff
@@ -20,12 +23,19 @@ type Space struct {
 }
 
 type spaceDoc struct {
+	DocId      string `bson:"_id"`
+	Id         string `bson:"spaceid"`
```

Comment:
This would also be simplified if we put Id => _id. I believe you get automatic load/save adding the prefix and removing the prefix as well.